library/core/src/test/java/com/google/android/exoplayer2/text/subrip/SubripDecoderTest.java

```diff
@@ -154,6 +155,27 @@ public void testDecodeNoEndTimecodes() throws IOException {
         .isEqualTo("Or to the end of the media.");
   }
 
+  @Test
+  public void testDecodeCueWithTag() throws IOException{
+    SubripDecoder decoder = new SubripDecoder();
+    byte[] bytes = TestUtil.getByteArray(RuntimeEnvironment.application, TYPICAL_WITH_TAGS);
+    SubripSubtitle subtitle = decoder.decode(bytes, bytes.length, false);
+    assertThat(subtitle.getCues(subtitle.getEventTime(0)).get(0).text.toString())
+        .isEqualTo("This is the first subtitle.");
+    assertThat(subtitle.getCues(subtitle.getEventTime(2)).get(0).text.toString())
+        .isEqualTo("This is the second subtitle.\nSecond subtitle with second line.");
+    assertThat(subtitle.getCues(subtitle.getEventTime(4)).get(0).text.toString())
+        .isEqualTo("This is the third subtitle.");
+
+    // Based on the SSA v4+ specs the curly bracket must be followed by a backslash, so this is
```

Comment:
I followed the docs that can be downloaded from here:
moodub.free.fr/video/ass-specs.doc

And it says: "All override codes are always preceded by a backslash \"
So that's why I didn't allow a space after the bracket. Regarding spaces after the backslash I just followed how the VLC handles these cases.

We sure can require no whitespace in the bracketed region, is up to you :)

Reply:
Guess we can allow space, if there's an existing example of doing this (as there seems to be)!